test/xpu_api/numerics/complex.number/cmplx.over/pow.pass.cpp

```diff
@@ -63,15 +63,15 @@ test(const dpl::complex<T>& x, const dpl::complex<U>& y)
 
 template <class T, class U>
 void
-test(typename std::enable_if<std::is_integral<T>::value>::type* = 0, typename std::enable_if<!std::is_integral<U>::value>::type* = 0)
```

Comment:
To be honest, I don't understand the logic of those functions. Let's simplify unless you have clear explanation why it should be written in that way.

Reply:
This was part of source test implementation which were imported into our code.
I don't think that we should fix any places in tests in this PR.

Reply:
Another argument to keep it as it is right now is that this is the way how it's implemented in LLVM libc++ tests. So, it's not our invention. I would probably prefer to simplify that, but I agree, not in this PR